### ISSUE TITLE
DND5e - Error occurred while rendering ActorSheet5eCharacter

### ISSUE THREAD

**Issue by UranusBytes** [League-of-Foundry-Developers/foundryvtt-devMode#56]
**Environment Details**
- Foundry Core Version:  10.285 (latest as of 9/10)
- System & Version: DND5e 2.0.2 (latest as of 9/10)
- Modules: Developer Mode 1.15.3   (latest as of 9/10)
- OS & Browser: Chrome/Multiple
- Hosting: Self Hosted

**Describe the bug**
Newly created world with a single scene, single actor.  Attempting to create a new actor or open character sheet for existing actor throws error in console.

```
{
    "message": "An error occurred while rendering ActorSheet5eCharacter 41. The partial dnd5e.actor-warnings could not be found",
    "name": "Error"
}
```

```
Foundry VTT | Retrieved and compiled template systems/dnd5e/templates/actors/character-sheet.hbs
foundry.js:711 d {description: undefined, fileName: undefined, lineNumber: undefined, endLineNumber: undefined, message: 'An error occurred while rendering ActorSheet5eChar…e partial dnd5e.actor-warnings could not be found', …}description: undefinedendLineNumber: undefinedfileName: undefinedlineNumber: undefinedmessage: "An error occurred while rendering ActorSheet5eCharacter 41. The partial dnd5e.actor-warnings could not be found"name: "Error"number: undefinedstack: "Error: An error occurred while rendering ActorSheet5eCharacter 41. The partial dnd5e.actor-warnings could not be found\n    at Object.h (http://localhost:30000/scripts/handlebars.min.js:27:18644)\n    at Object.c [as invokePartial] (http://localhost:30000/scripts/handlebars.min.js:27:15036)\n    at Object.eval [as main] (eval at createFunctionContext (http://localhost:30000/scripts/handlebars.min.js:29:1808), <anonymous>:17:28)\n    at c (http://localhost:30000/scripts/handlebars.min.js:27:15429)\n    at d (http://localhost:30000/scripts/handlebars.min.js:27:15744)\n    at e (http://localhost:30000/scripts/handlebars.min.js:28:22748)\n    at renderTemplate (http://localhost:30000/scripts/foundry.js:6870:10)\n    at async ActorSheet5eCharacter._renderInner (http://localhost:30000/scripts/foundry.js:5189:16)\n    at async ActorSheet5eCharacter._renderInner (http://localhost:30000/scripts/foundry.js:5750:18)\n    at async ActorSheet5eCharacter._render (http://localhost:30000/scripts/foundry.js:5020:19)"[[Prototype]]: Error
    at Object.<anonymous> (http://localhost:30000/scripts/handlebars.min.js:27:5241)
    at b (http://localhost:30000/scripts/handlebars.min.js:27:330)
    at Object.<anonymous> (http://localhost:30000/scripts/handlebars.min.js:27:2069)
    at b (http://localhost:30000/scripts/handlebars.min.js:27:330)
    at Object.<anonymous> (http://localhost:30000/scripts/handlebars.min.js:27:1435)
    at b (http://localhost:30000/scripts/handlebars.min.js:27:330)
    at Object.<anonymous> (http://localhost:30000/scripts/handlebars.min.js:27:792)
    at b (http://localhost:30000/scripts/handlebars.min.js:27:330)
    at http://localhost:30000/scripts/handlebars.min.js:27:417
    at http://localhost:30000/scripts/handlebars.min.js:27:422
onError @ foundry.js:711
(anonymous) @ foundry.js:4975
Promise.catch (async)
render @ foundry.js:4973
render @ foundry.js:6311
_onClickDocumentName @ foundry.js:60167
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
foundry.js:5005 Foundry VTT | Rendering ActorSheet5eCharacter
```

Disabling module lets character sheet be rendered

**To Reproduce**
1. Attempt to open actor character sheet from scene or actor list

**Expected behavior**
Open actor character sheet

**Screenshots**
<img width="614" alt="image" src="https://user-images.githubusercontent.com/11236319/189484933-d2ef6fe7-ebe5-49a2-859d-18aa08ad5db5.png">

**Comment by akrigline** [comment]
probably related to #55
Just released 1.15.4, See if that fixes this.

**Comment by UranusBytes** [comment]
Confirmed that 1.15.4 resolves this issue.  Thanks!
